hil-test/tests/esp_wifi_ble_controller.rs

```diff
@@ -0,0 +1,68 @@
+//! Test we can talk to the BLE controller.
+
+//% CHIPS: esp32 esp32c2 esp32c3 esp32c6 esp32h2 esp32s3
+//% FEATURES: unstable esp-wifi esp-alloc esp-wifi/ble
+
+#![no_std]
+#![no_main]
+
+use embedded_io::{Read, Write};
+use hil_test as _;
+
+#[cfg(test)]
+#[embedded_test::tests(default_timeout = 3)]
+mod tests {
+    use esp_hal::{clock::CpuClock, peripherals::Peripherals, rng::Rng, timer::timg::TimerGroup};
+    use esp_wifi::ble::controller::BleConnector;
+
```

Comment:
idk if it's intentional but in other HIL tests the imports go at the top level (i.e. lines 9-10)